### PR TITLE
ENG-3763 Base mfe-config path on a microfrontend's publicFolder

### PR DESCRIPTION
**Pull request by raxkaynan**

### CODE REVIEW

File: test/commands/api/add.test.ts [hunk 1]
```diff
@@ -131,7 +133,9 @@ describe('api add', () => {
 
   test
     .do(() => {
-      fs.rmSync(path.resolve('microfrontends', 'mfe1', 'mfe-config.json'))
+      fs.rmSync(
+        path.resolve('microfrontends', 'mfe1', 'public', 'mfe-config.json')
```
Comment (cecchisandrone) [inline test/commands/api/add.test.ts]: use constant for mfe-config.json

**Review by cecchisandrone — CHANGES_REQUESTED** [review]

File: test/commands/api/add.test.ts (hunk 1 above)
Comment (raxkaynan) [reply test/commands/api/add.test.ts]: that's just for the test. not sure if it's necessary to use a source constant there.

Comment (cecchisandrone) [reply test/commands/api/add.test.ts]: as we already have it defined probably worth to be used

Comment (raxkaynan) [reply test/commands/api/add.test.ts]: just changed `getMfeConfigPath()` from `private` to `public` instead so it can be used there

**Review by cecchisandrone — APPROVED** [review]

**Pull request merged**